### PR TITLE
Update index page and nas quotas page to bootstrap 5

### PR DESCRIPTION
**Pull request by aanil**
I thought it'll be good to have this update before someone works on changing the Index page.

### CODE REVIEW

File: run_dir/design/index.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{% extends "base.html" %}
+{% extends "base_b5.html" %}
 
 {% block stuff %}
 <h1 id="scilifelabgenomics-status">SciLifeLab Genomics - Status</h1>
```
Comment (alneberg) [inline run_dir/design/index.html:4]: I would add `class="mt-4"` here or similar.

File: run_dir/design/index.html [hunk 2]
```diff
@@ -13,39 +13,39 @@ <h2> Get Access </h2>
...
     </div>
 </form>
 
-<div class="row homepage-ctas">
+<div class="row homepage-ctas mt-3 justify-content-md-center">
 	<div class="col-lg-offset-3 col-lg-2 col-sm-4 col-xs-6">
 		<div class="well">
-			<h3><a href="/projects"><i class="glyphicon glyphicon-search"></i></a></h3>
+			<h3><a href="/projects"><i class="fa fa-search"></i></a></h3>
 			<p><a class="btn btn-primary" href="/projects">View All Projects</a></p>
```
Comment (alneberg) [inline run_dir/design/index.html:28]: I think all three of thes buttons would look better with the `btn-lg` class.

File: run_dir/design/index.html [hunk 3]
```diff
@@ -13,39 +13,39 @@ <h2> Get Access </h2>
...
 		<div class="well">
-			  <h3><a href="https://ngi-dashboards.scilifelab.se/dbe/"><i class="glyphicon glyphicon-globe"></i></a></h3>
+			  <h3><a href="https://ngi-dashboards.scilifelab.se/dbe/"><i class="fa fa-globe-europe"></i></a></h3>
 			<p><a class="btn btn-primary"  href="https://ngi-dashboards.scilifelab.se/dbe/">External Dashboard</a></p>
 		</div>
 	</div>
 </div>
 
 <div class="alert alert-info">
     <p style="text-align:center;">Want to see something different here? Let us know on the
```
Comment (alneberg) [inline run_dir/design/index.html:46]: Maybe a `class="mb-0"` here?

File: run_dir/design/index.html [hunk 4]
```diff
@@ -54,7 +54,7 @@ <h3><a href="https://ngi-dashboards.scilifelab.se/dbe/"><i class="glyphicon glyp
 		<h3>Server Status</h3>
```
Comment (alneberg) [inline run_dir/design/index.html:54]: And finally I would throw in a `class="mt-5"` on the row div above.

File: run_dir/design/index.html (hunk 1 above)
Comment (aanil) [reply run_dir/design/index.html]: Done!

File: run_dir/design/index.html (hunk 2 above)
Comment (aanil) [reply run_dir/design/index.html]: Done!

File: run_dir/design/index.html (hunk 3 above)
Comment (aanil) [reply run_dir/design/index.html]: Done!

File: run_dir/design/index.html (hunk 4 above)
Comment (aanil) [reply run_dir/design/index.html:54]: Done!

**Review by alneberg — APPROVED** [review]
Thank you!

**Pull request merged**